src/widgets/ErrorDialog.h

```diff
@@ -54,6 +54,13 @@ void ShowErrorDialog(wxWindow *parent,
                      bool Close = true,
                      const wxString &log = {});
 
+/// Displays an error dialog, possibly alowing to send error report
+AUDACITY_DLL_API
+void ShowExceptionDialog(
+   wxWindow* parent, const TranslatableString& dlogTitle,
+   const TranslatableString& message, const wxString& helpPage,
+   bool Close = true, const wxString& log = {});
+
```

Comment:
You added a function to this file, but I had deleted this file in a branch

I rewrite my branch so I don't delete the file, but I restrict inclusion of it to just one place.

This needs to be hidden behind the BasicUI facade at some point, so lib-exceptions and others can avoid dependency on wxCore.